ui/StatusQ/src/StatusQ/Core/StatusAssetSettings.qml

```diff
@@ -9,6 +9,7 @@ QtObject {
     property url source
     property real width
     property real height
+    property real scale
```

Comment:
This is not used

I know there's a warning somewhere about it, so just remove the usage

Reply:
fixed